tensorflow/core/common_runtime/dml/dml_device_context.cc

```diff
@@ -148,4 +121,151 @@ void DMLDeviceContext::CopyDeviceTensorToCPU(const Tensor* device_tensor,
   event_queue_->Enqueue(status_or_event.ConsumeValueOrDie(), callback);
 }
 
+// TODO: consider moving this code into EC outside of lock
```

Comment:
This TODO could easily get lost. If we don't want to create a tracking bug/task for it, maybe we should at least make it `TODO (tfdml): ...` (same for the one below)

Reply:
This change has enough refactoring as is so I'm just going to delete the TODO. There's no practical benefit to this anyway.